src/applications/ivc-champva/10-7959f-1/config/form.js

```diff
@@ -60,21 +63,26 @@ const formConfig = {
   defaultDefinitions: {},
   chapters: {
     applicantInformationChapter: {
-      title: 'Applicant Information',
+      title: `Veteran's Information`,
       pages: {
         page1: {
           path: 'veteran-information',
-          title: 'Veteran Information',
+          title: 'Veteran Personal Information',
           uiSchema: {
-            fullNameTitle: inlineTitleUI('Your name'),
+            ...titleUI(
+              `Veteran's personal information`,
+              `We use this information to contact you and verify other details.`,
+            ),
             fullName: fullNameUI(),
+            veteranDOB: dateOfBirthUI(),
           },
           schema: {
             type: 'object',
```

Comment:
It's a nit, but can these back ticks be changed to quotes?

Reply:
Okay. Are back ticks not allowed (just for my future personal knowledge)